Mage.Tests/src/test/java/org/mage/test/serverside/base/impl/CardTestPlayerAPIImpl.java

```diff
@@ -1572,6 +1572,23 @@ public void assertIsAttachedTo(TestPlayer thePlayer, String theAttachment, Strin
                                 .collect(Collectors.toList()).contains(theAttachment)));
 
 
+    }
+
+    public void assertIsNotAttachedTo(TestPlayer thePlayer, String theAttachment, String thePermanent) {
```

Comment:
This can be a separate scope, but probably we should refactor to a single method `assertAttachedTo` that takes a boolean parameter